multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/template/TemplateFactory.java

```diff
@@ -117,8 +119,154 @@ public final void fillTemplateModelClassException(final Path filePathToSave, fin
     pathToExceptionPackage.toFile().mkdirs();
     root.put("packageModel", modelPackage);
     final String pathToSaveMainClass = pathToExceptionPackage.resolve("ModelClassException.java").toString();
-    writeTemplateToFile(com.sngular.api.generator.plugin.openapi.template.TemplateIndexConstants.TEMPLATE_MODEL_EXCEPTION, root, pathToSaveMainClass);
+    writeTemplateToFile(TemplateIndexConstants.TEMPLATE_MODEL_EXCEPTION, root, pathToSaveMainClass);
+  }
+
+  public final void fillTemplateNotNull(final Path filePathToSave, final String modelPackage) throws IOException, TemplateException {
```

Comment:
Review those methods, it seems that it just change the value of `"NotNull.java"` and `TemplateIndexConstants.TEMPLATE_NOT_NULL_ANNOTATION`.
May be creating a method which receives the value of those two constants, we can unify the implementation of methods in L.125, 133, 141, 149...

Reply:
Done

Reply:
Done